File: Problemset/number-of-good-pairs/number-of-good-pairs.py

# @Title: 好数对的数目 (Number of Good Pairs)
# @Author: 846188037@qq.com
# @Date: 2020-07-12 21:06:58
# @Runtime: 48 ms
# @Memory: 13.5 MB

class Solution:
    def numIdenticalPairs(self, nums: List[int]) -> int:
        count=0
        for i in range(len(nums)-1):
            for j in range(i+1,len(nums)):
                if nums[i]==nums[j]:
                    count+=1
        return count

File: Problemset/number-of-good-pairs/number-of-good-pairs.cs

// @Title: 好数对的数目 (Number of Good Pairs)
// @Author: 846188037@qq.com
// @Date: 2020-07-13 00:30:34
// @Runtime: 100 ms
// @Memory: 23.5 MB

public class Solution {
    public int NumIdenticalPairs(int[] nums) {
        int count=0;
        for(int i=0;i<nums.Length-1;i++){
            for(int j=i+1;j<nums.Length;j++){
                if(nums[i]==nums[j]){
                    count++;
                }
            }
        }
        return count;

    }
}
